routes/botSetting.js

```diff
@@ -0,0 +1,68 @@
+const express = require("express")
+const botSetting = require("../db/botSetting.js")
+const router = express.Router()
+require('dotenv').config();
```

Comment:
I still see that sometimes you use semicolons and sometimes do not. Update your eslint config to require them